warehouse/macros/make_schedule_file_dimension_from_dim_schedule_feeds.sql

```diff
@@ -1,12 +1,39 @@
 {% macro make_schedule_file_dimension_from_dim_schedule_feeds(dim_schedule_feeds, gtfs_file_table) %}
+{{ config(materialized='incremental') }}
```

Comment:
It feels a bit weird to me to have the materialization in a macro. I guess it is ok? But I am a bit 🤔

Reply:
Yeah it's a bit weird but ideally we just force anything using this macro to be incremental, since the macro itself only works in an incremental model.

Multiple calls to config are explicitly supported, luckily https://github.com/dbt-labs/dbt-core/pull/1150